pkg/spec/spec.go

```diff
@@ -31,14 +31,17 @@ type Function struct {
 
 // FunctionSpec contains func specification
 type FunctionSpec struct {
-	Handler  string             `json:"handler"`
-	Function string             `json:"function"`
-	Runtime  string             `json:"runtime"`
-	Type     string             `json:"type"`
-	Topic    string             `json:"topic"`
-	Schedule string             `json:"schedule"`
-	Deps     string             `json:"deps"`
-	Template v1.PodTemplateSpec `json:"template" protobuf:"bytes,3,opt,name=template"`
+	Handler     string             `json:"handler"`      // Function handler: "file.function"
```

Comment:
How does this `Handler` prefix relate to the new `File` value ?

Reply:
`File` is the filename in the origin. `Handler` contains the filename in the destination + the function to load. Some examples:
 - `File: "function.zip", Handler: "hello.get"`: The origin file is function.zip that will get extracted. Once it is extracted it will load the function `get` from the file `hello.py`
 - `File: "function.py", Handler: "hello.get"`: The origin file is function.py so we don't need to preprocess it. Then is renamed as `hello.py` and it will load the function `get`. I am not very happy with this version of the workflow since the "handler" name is useless and could be confusing but it was that way before so to avoid breaking changes we should leave it as it is for the moment.